### ISSUE TITLE
Comments are not removed as expected

### ISSUE THREAD

**Issue by adammockor** [digisfera/useref#41]
Hello,

in version 0.3.11 this code:
```
<!-- build:js(./app/) scripts/vendor.min.js-->
<!-- bower:js-->
<script src="../bower_components/jquery/dist/jquery.js"></script>
<script src="../bower_components/bootstrap-sass/assets/javascripts/bootstrap.js"></script>
<!-- endbower-->
<!-- endbuild-->
```
is compiled to this:
```
<script src="scripts/vendor.min.js-->" <!-- bower:js></script>
```

Version 0.3.10 works fine. (node: 0.12.1, os x 10.10)

**Comment by adammockor** [comment]
Was this fixed in 0.3.12? Looks like it was.

**Comment by manuelcabral** [comment]
Hi,

Yes, #43 fixed this.
